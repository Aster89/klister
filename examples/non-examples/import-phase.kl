#lang kernel

-- This should fail because the only thing imported at phase 1 is "import".

[meta
  [define m-impl
    [lambda [s]
      (syntax-case s
        [[vec [_ e]] [pure e]])]]]

[define-macros ([m m-impl])]

[example [m 1]]
