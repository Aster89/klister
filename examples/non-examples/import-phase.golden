Unknown: #[import-phase.kl:6.4-6.10]<define>
